pkg/cloudevents/transport/pubsub/internal/connection.go

```diff
@@ -36,6 +36,8 @@ type Connection struct {
 	subWasCreated  bool
 	subOnce        sync.Once
 
+	ReceiveSettings *pubsub.ReceiveSettings
```

Comment:
add a comment?

Reply:
will these be configurable when setting up the transport?

Reply:
Missed that. Added now.